service/frontend/workflow_handler.go

```diff
@@ -99,6 +99,9 @@ const (
 	errTooManyDeleteDeploymentRequests  = "Too many DeleteWorkerDeployment requests have been issued in rapid succession. Please throttle the request rate to avoid exceeding Worker Deployment resource limits."
 	errTooManyDeleteVersionRequests     = "Too many DeleteWorkerDeploymentVersion requests have been issued in rapid succession. Please throttle the request rate to avoid exceeding Worker Deployment resource limits."
 	errTooManyVersionMetadataRequests   = "Too many UpdateWorkerDeploymentVersionMetadata requests have been issued in rapid succession. Please throttle the request rate to avoid exceeding Worker Deployment resource limits."
+
+	maxReasonLength   = 256 // Maximum length for the reason field in RateLimitUpdate configurations.
+	maxIdentityLength = 256 // Maximum length for the identity field in UpdateRateLimitRequest
```

Comment:
elsewhere in this file I see a lot of checks against `wh.config.MaxIDLengthLimit()`, so might as well use that one for identity to be consistent